jest.setup.js

```diff
@@ -1 +1 @@
-jest.setTimeout(30000)
+jest.setTimeout(9999999)
```

Comment:
@shrijan-siris I think we shouldn't increase beyond 30 seconds. A single test shouldn't take more than 10 seconds.

Reply:
we should get a way to test and mock for mongoose call, i was testing with 2 real data and took more than 30 seconds